mods/persistence/modules/mob/new_player.dm

```diff
@@ -217,7 +229,7 @@
 		new_db_connection = TRUE
 
 	spawning = TRUE
-	var/DBQuery/char_query = dbcon_save.NewQuery("SELECT `key` FROM `limbo` WHERE `type` = '[LIMBO_MIND]' AND `metadata` = '[key]' AND `metadata2` = '[selected_char_name]'")
+	var/DBQuery/char_query = dbcon_save.NewQuery("SELECT `key` FROM `limbo` WHERE `type` = '[LIMBO_MIND]' AND `metadata` = '[sanitize_sql(key)]' AND `metadata2` = '[sanitize_sql(selected_char_name)]'")
```

Comment:
Just a comment, but we really should keep db queries all at the same place. Having all that stuff all over the code is a nightmare for maintenance.

Reply:
I agree. I also greatly dislike this whole `metadata` and `metadata2` business so I'd like to solve that simultaneously if we can.